angel_system/uho/src/data_helper.py

```diff
@@ -1,26 +1,51 @@
 """
 Helper functions for forming the input data to the UHO model.
 """
+from dataclasses import dataclass
 from typing import Callable
 from typing import Dict
 from typing import List
+from typing import Tuple
 
 from angel_msgs.msg import ObjectDetection2dSet
 import numpy as np
 import torch
 
+
+@dataclass
+class AuxData:
+    """
+    Class for representing the aux data dictionary required by the UHO module.
```

Comment:
We could add a note that it is expected that all list attributes are expected to be the same length.